### PR TITLE
Create Formulaic Report a Violation

### PR DESCRIPTION
**Pull request by aroybalreid**
Please create a page for Formulaic to share how users can report a violation.  There is not a URL for this page yet, but we will have one created once the document is put together.  Note that in line 33, "Terms of Service" and "Content Policies" will eventually need to include links to the pages with the Formulaic TOS and Content Policies.

### CODE REVIEW

**Comment by aroybalreid** [comment]
No localization needed.  English only at this time.

**Comment by flodolo** [comment]
As explained in previous pull requests, you cannot create a file without an extension.

`en/Formulaic Report a Violation` needs to include `.md`, no space, and be lowercase.

**Comment by flodolo** [comment]
Other problems:
* Lack of empty lines after a title.
* Lack of space before and after lists, and after the asterisk in bullet items.

Fixed in https://github.com/mozilla/legal-docs/pull/2031/commits/1adb9546e7abd9ae1970248dde542b0aa650dc87

Please check the [current preview](https://github.com/mozilla/legal-docs/blob/1adb9546e7abd9ae1970248dde542b0aa650dc87/en/formulaic_report_violation.md) to make sure it matches your original expectations

File: en/formulaic_report_violation.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Report a Violation
```
Comment (flodolo) [inline en/formulaic_report_violation.md:1]: Shouldn't we include Formulaic in the title of the page, e.g. `Formulaic — Report a violation` or similar?

File: en/formulaic_report_violation.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Our moderation team will review your content violation report and make a decision. If we deem that the reported Formula is in violation, we will unpublish it from the Formulaic website and notify the publisher. The Formula publisher has six months in which to appeal our decision.
+
+We will notify you of our content decision at the same email address that you used to submit your report.
+
+Depending on the severity of a violation, we may take further action against the Formula publisher, including terminating their Formulaic account.
+
+# Publishers: What you can expect if your Formula is flagged for review
+
+All Formulaic users are bound by our terms of service and our content policies. If your Formula is flagged for review and our moderators decide that your Formula does not meet our guidelines, we will unpublish it from the public website. We will then notify you via email that you are in violation of the content policies.
```
Comment (flodolo) [inline en/formulaic_report_violation.md:39]:
```suggestion
All Formulaic users are bound by our terms of service and our content policies. If your Formula is flagged for review and our moderators decide that your Formula does not meet our guidelines, we will remove it from the public website. We will then notify you via email that you are in violation of the content policies.
```

"unpublish from the public website" feels repetitive?

**Comment by flodolo** [comment]
Made more changes to the header levels, updated [preview here](https://github.com/mozilla/legal-docs/blob/103fff67eefeeb37c66c9a6712da74c384ddf08b/en/formulaic_report_violation.md).

Only the main title should be H1 (`#`), and the rest didn't seem to follow the logical hierarchy.

File: en/formulaic_report_violation.md (hunk 1 above)
Comment (aroybalreid) [reply en/formulaic_report_violation.md:1]: We could say "Report a Violation on Formulaic".  For the time being, the plan is that this page would only be accessible from Formulaic, so it should be clear which product this relates to.  But I don't see any reason not to make it clearer.

File: en/formulaic_report_violation.md (hunk 2 above)
Comment (aroybalreid) [reply en/formulaic_report_violation.md:39]: Remove is fine.

**Comment by aroybalreid** [comment]
Sorry for the naming errors @flodolo, and thanks for the corrections.  I reviewed the preview, and it looks good.  I'm happy with the title change discussed ("Report a Violation on Formulaic")

**Review by flodolo — APPROVED** [review]

**Pull request merged**